benchmark/performance_utils.py

```diff
@@ -424,7 +424,7 @@ def set_more_shapes(self):
 
 
 def generate_tensor_input(shape, dtype, device):
-    if dtype in FLOAT_DTYPES:
+    if dtype in FLOAT_DTYPES + [torch.float64]:
```

Comment:
I think we don’t need to focus on `torch.float64` accuracy for this case. In real-world model training and inference scenarios, `float64` precision is rarely used.